atmo/jobs/migrations/0016_assign_group_view_perm.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-20 23:26
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from django.db import migrations
+from guardian.shortcuts import assign_perm, remove_perm
+
+
+def assign_spark_job_view_permission_to_group(apps, schema_editor):
+    SparkJob = apps.get_model('jobs', 'SparkJob')
+    group, _ = Group.objects.get_or_create(name='Spark job maintainers')
+    assign_perm('view_sparkjob', group, SparkJob.objects.all())
```

Comment:
The last time I did that assign_perm did not work correctly inside the Python migration step. See the `PermissionMigrator` class in `atmo/models.py` as an alternative, which also makes sure the permission exists at the time of the migration run. Could you modify the class to offer a way to assign permissions to groups in addition to users and use that here please?